Readme.md

```diff
@@ -191,9 +190,7 @@ const connection = Client.connect({
 In practice you probably would want client/server communication to happen across the wire, or at least across processes. You can bring your own transport channel, or choose an existing one. For example:
 
 ```ts
-import * as HTTP from '@ucanto/transport/http'
-import * as CAR from '@ucanto/transport/car'
-import * as CBOR from '@ucanto/transport/cbor'
+import Transport from '@ucanto/transport'

```

Comment:
This would require making the Transport directory have an index.js

Reply:
it would also require it to have a `default` export